tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -1281,6 +1281,13 @@ class CompletionSuite extends BaseCompletionSuite {
            |implicitConversions scala.languageFeature
            |postfixOps scala.languageFeature
            |""".stripMargin,
+      "2.13.12" ->
```

Comment:
We should probably add ranges to compat version, otherwise it will get annoying. But that can be done later.